eng/pipelines/typespec-apiview.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
+  - name: APIViewArtifactsDirectoryName
+    type: string
+    default: 'TypespecAPIViewArtifacts'
+  - name: APIViewArtifactsName
+    type: string
+    default: 'typeSpecAPIViewArtifacts'
+  - name: APIViewAPIUri
+    type: string
+    default: 'https://apiviewstagingtest.com/PullRequest/DetectAPIChanges'
+    # 'https://apiviewstagingtest.com/PullRequest/DetectAPIChanges'
+    # 'https://apiview.dev/PullRequest/DetectAPIChanges'
```

Comment:
```suggestion
    default: 'https://apiview.dev/PullRequest/DetectAPIChanges'
    # 'https://apiviewstagingtest.com/PullRequest/DetectAPIChanges'
```

Reply:
it's staging now for testing purposes

Reply:
Did we need to use staging because this requires a change on the APIView server side?

Reply:
No, we’re using staging to compare this and the old alps pipeline run (due to racing condition, only one copy of the apiview will b generated by either this or alps pipeline